app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModel.kt

```diff
@@ -19,7 +19,9 @@
 package org.kiwix.kiwixmobile.zim_manager
 
 import android.app.Application
+import android.content.Context
```

Comment:
static analysis failed on this unused import.

Reply:
I'll fix that in the next commit. There is a minor issue in the pr right now. Hopefully, I'll fix both the issues with the next commit.